### PR TITLE
Fix `typing_extensions` ImportError on Windows

### PR DESCRIPTION
**Pull request by T-256**
Split out of https://github.com/yifeikong/curl_cffi/pull/176

Missed dependency for #172

### CODE REVIEW

**Comment by T-256** [comment]
This PR is start of merge chain that split out from https://github.com/yifeikong/curl_cffi/pull/176.

Branches order from [my fork](https://github.com/T-256/curl_cffi) is:
> proactor -> cffi-backend -> sync-docs -> certifi -> libcurl -> no-include

When each branch merged, I'll open next PR.

File: preprocess/download_so.py [hunk 1]
```diff
@@ -15,19 +13,6 @@
 else:
     LIBDIR = "/usr/local/lib"
 
-
```
Comment (yifeikong) [inline preprocess/download_so.py:18]: GitHub can be extremely slow in some countries, like China, where I live in. This tiny progress indicator can make people know that it does not hang.

Comment (T-256) [reply preprocess/download_so.py:18]: This process would take on server side, I think don't showing long output of progress could help you at there.
Note that file < 10MB took less than a second on GH Actions.

I encountered its anonymity while watching live ci outputs.

Comment (T-256) [reply preprocess/download_so.py:18]: We can add it for now, and if it causes any problems, we can revert it any time. it's not really about the main codebase.

Comment (yifeikong) [reply preprocess/download_so.py:18]: Sometimes it's needed to run locally, not on github actions, can we put it behind an `if` or something.

**Pull request merged**